com.unity.cinemachine/Editor/Editors/InputAxisControllerEditor.cs

```diff
@@ -131,20 +91,18 @@ static DefaultControlInitializer()
                     if (axis.Name.Contains("Move"))
                     {
                         actionName = "Player/Move";
-                        inputName = axis.AxisIndex == 0 ? "Horizontal" : (axis.AxisIndex == 1 ? "Vertical" : "");
-                        controller.Recentering = new InputAxisRecenteringSettings { Enabled = true };
+                        inputName = axis.Hint == IInputAxisSource.AxisDescriptor.Hints.X ? "Horizontal" 
+                            : (axis.Hint == IInputAxisSource.AxisDescriptor.Hints.Y ? "Vertical" : "");
```

Comment:
This could be also written as 
```
inputName = axis.Hint switch
{
           IInputAxisSource.AxisDescriptor.Hints.X => "Horizontal",
           IInputAxisSource.AxisDescriptor.Hints.Y => "Vertical",
           _ => ""
};
```